web/cobrands/fixmystreet/staff.js

```diff
@@ -342,3 +342,68 @@ $.extend(fixmystreet.set_up, {
     });
   }
 });
+
+fixmystreet.maps = fixmystreet.maps || {};
+
+$.extend(fixmystreet.maps, {
+  shortlist_multiple: function(ids, token, count) {
+    var retryCount = (typeof count !== 'undefined') ?  count : 0;
+    $.post("/my/planned/change_multiple", { ids: ids, token: token })
+    .done(function() {
+      var $itemList = $('.item-list'),
+          items = [];
+
+      for (var i = 0; i < ids.length; i++) {
+        var problemId = ids[i],
+            $item = $itemList.find('#report-'+ problemId),
+            $form = $item.find('form'),
+            $submit = $form.find("input[type='submit']" );
+
+        $item.attr('data-lastupdate', new Date().toISOString());
```

Comment:
Repeating from previous review:
dracos: I'm not clear on what this is doing. lastupdate isn't usually updated on an add/remove shortlist, because the report hasn't changed. The lastupdate functionality should still operate okay without this, if the report is newly added, I think?
pezholio: I'm just following the conventions that were applied on a single update a la: https://github.com/mysociety/fixmystreet/blob/master/web/cobrands/fixmystreet/offline.js#L413-L416
dracos: That caches the lastupdate, yes, but it doesn't change it, as this code does?